webroot/src/pages/LicenseeDashboard/LicenseeDashboard.ts

```diff
@@ -120,6 +105,13 @@ export default class LicenseeDashboard extends Vue {
         });
     }
 
+    viewLicenseeProof() {
```

Comment:
Would this name make more sense for consistency, or do the names differ intentionally?

```suggestion
    viewLicenseeVerification() {
```